app/reader/filters.py

```diff
@@ -5,8 +5,9 @@
 @reader_bp.app_template_filter('display_date')
 def display_date(dt):
     diff = datetime.now() - dt
```

Comment:
I've always been a fan of variable names that aren't abbreviated, would have gone with something like time_elapsed = datetime.now() - dt

Reply:
Agreed. We could probably just look to remove the abbreviations everywhere for clarity.

Reply:
Sounds good, dude this is awesome, GitHub rocks.